src/relax/transform/fuse_tir.cc

```diff
@@ -897,8 +973,11 @@ class TIRFuseMutator : public ExprMutator {
     for (const auto& [gv, func] : mod->functions) {
       // Only fuse primitive relax functions
       if (func->IsInstance<relax::FunctionNode>() && func->HasNonzeroAttr(attr::kPrimitive)) {
-        tir::PrimFunc fused_tir = FusedTIRConstructor::GetFusedTIR(mod, gv);
-        mutator.fused_tir_funcs_.Set(gv, fused_tir);
+        const auto& [prim_func, indices] = FusedTIRConstructor::GetFusedTIR(mod, gv);
+        mutator.fused_tir_funcs_.Set(gv, prim_func);
+        if (!indices.empty()) {
```

Comment:
Nitpick: Using `if (indices.size())` instead of `if (!indices.empty())` would avoiding double-negatives and make the condition easier to read.  (Though, personal preference as `if (indices.size())` relies on conversion of non-zero `size_t` to `true`.)

Reply:
Kind of subjective, I think "not empty" is readable (sounds more like how it would be phrased verbally). I'll change it if you prefer it.

Reply:
Probably more personal preference, so either works.